Dalamud/Game/ClientState/Objects/Enums/StatusFlags.cs

```diff
@@ -34,23 +34,23 @@ public enum StatusFlags : byte
         OffhandOut = 8,
 
         /// <summary>
-        /// Character is a party member.
+        /// Character is crafting.
         /// </summary>
-        PartyMember = 16,
+        Crafting = 16,
 
         /// <summary>
-        /// Character is a alliance member.
+        /// Character is a party member.
         /// </summary>
-        AllianceMember = 32,
+        PartyMember = 32,
 
         /// <summary>
-        /// Character is in friend list.
+        /// Character is a alliance member.
         /// </summary>
-        Friend = 64,
+        AllianceMember = 64,
 
         /// <summary>
-        /// Character is casting.
+        /// Character is in friend list.
         /// </summary>
-        IsCasting = 128,
```

Comment:
Why is IsCasting gone? Is it crafting instead now?
This would break the API.

Reply:
For whatever odd reason, it seems gone, and everything from PartyMember onwards has been shifted.
What was PartyMember was lighting up when people were crafting, so assuming it's that.

Reply:
I was looking at this just now and noticed that one of my friends had the flag of `IsCasting` and nothing else set to true. It looks like everything got shifted?